### PR TITLE
[FIX] product_template_tags: fix rule because multicompany scheme change

### PR DESCRIPTION
**Pull request by jcadhoc**

### CODE REVIEW

**Review by jjscarafia — APPROVED** [review]
LGTM

File: product_template_tags/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Product Template Tags",
     "summary": "This addon allow to add tags on products",
-    "version": "13.0.2.0.0",
+    "version": "13.0.2.1.0",
```
Comment (simahawk) [inline product_template_tags/__manifest__.py:7]: you should not change version, the bot will do it. Also, this is a patch not a feature ;)

**Review by simahawk — APPROVED** [review]
could you drop the version change?

**Comment by rousseldenis** [comment]
@jcadhoc

**Pull request merged**